inspirehep/records/api/base.py

```diff
@@ -273,11 +273,13 @@ def create(cls, data, **kwargs):
             record = super().create(data, id_=id_, **kwargs)
         return record
 
-    def get_linked_pids_from_field(self, path):
+    @classmethod
+    def _get_linked_pids_from_field(cls, data, path):
```

Comment:
TBH it's very weird to have  a ``classmethod`` starting with ``_`` and use it everywhere

Reply:
Actually this one is used only in the base class, but `_get_linked_records_from_field` is used in serializers, so propably I should change it's name....